app/assets/stylesheets/_base.scss

```diff
@@ -0,0 +1,8 @@
+html{
+  margin: 0;
+  padding: 0;
+}
+
+body{
+  padding: 100px 20px 100px 20px;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `100px 20px` instead of `100px 20px 100px 20px`